src/main/java/com/github/vkorobkov/jfixtures/util/YmlUtil.java

```diff
@@ -1,20 +1,43 @@
 package com.github.vkorobkov.jfixtures.util;
 
+import lombok.val;
 import org.yaml.snakeyaml.Yaml;
 
+import java.io.File;
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.util.Collections;
 import java.util.Map;
+import java.util.stream.Stream;
 
 public final class YmlUtil {
+    private static final String YML_EXT = ".yml";
+    private static final String YAML_EXT = ".yaml";
+
     private YmlUtil() {
     }
 
     @SuppressWarnings("unchecked")
     public static Map<String, Object> load(Path file) throws IOException {
         Object loaded = new Yaml().load(Files.newInputStream(file));
-        return loaded == null ? Collections.emptyMap() : (Map<String, Object>)loaded;
+        return loaded == null ? Collections.emptyMap() : (Map<String, Object>) loaded;
+    }
+
+    public static boolean hasYamlTwin(Path yamlFilePath) {
+        val fileCount = Stream.of(YML_EXT, YAML_EXT)
+                .map(ext -> cutOffExtension(yamlFilePath.toString()) + ext)
+                .filter(name -> new File(name).exists())
+                .count();
+
+        if (fileCount == 0) {
+            throw new IllegalArgumentException("Neither " + YAML_EXT + " nor " + YML_EXT + " file extension.");
```

Comment:
It's good when `IllegalArgumentException` contains bad argument name(`yamlFilePath` in our case)